documentation/settings/topics/integrations.rst

```diff
@@ -5,214 +5,13 @@
 Integrations
 ************
 
-Hyperview Integrations are a separately licensed set of features that let you push or pull data from external systems such as ServiceNow. Each integration is separately licensed.
+Hyperview APIs are documented from within the product and on the Hyperview Documentation site. Customers are free to write their integrations and there are API integration Samples provided on `GitHub <https://github.com/HyperviewHQ/API-Samples>`_.
```

Comment:
Should be a lowercase "s" in, "API integration Samples".

Reply:
Fixed. Thanks!